### PR TITLE
docs: add release notes for datasync

### PR DESCRIPTION
**Pull request by ssd71**
First time doing this :P

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ Please follow individual releases for more information.
   - [#1625](https://github.com/aerogear/graphback/issues/1625) Filters on "id: ID" field not working
   
   > NOTE: If you are migrating from [0.15](#0150) or previous versions of Graphback, you may be required to update relationship fields so that their values (previous stored as `String`) are of type [`ObjectID`](https://docs.mongodb.com/manual/reference/method/ObjectId/).
+  
+#### DataSync
```
Comment (machi1990) [inline docs/releases.md:22]: I think we can remove this.

File: docs/releases.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ Please follow individual releases for more information.
...
+
+* Using a `_lastUpdatedAt` field with a proper `GraphbackTimestamp` scalar and other minor fixes ([#1771](https://github.com/aerogear/graphback/pull/1771)) including:
+
+  - disabling conflicts in default configuration
+  - adding a `limit` argument to `sync` Queries
```
Comment (machi1990) [inline docs/releases.md:27]: Can we link fixed issues if any?

**Comment by ssd71** [comment]
@machi1990 Thanks for the review! will push something tomorrow as this one is already merged

**Comment by machi1990** [comment]
> @machi1990 Thanks for the review! will push something tomorrow as this one is already merged

My pleasure. This was good work, simple and straight to the point. In regard to my comments, it is okay, just minor improvements. Feel free to ignore them or push some changes in another PR that'll modify the release notes file.

**Pull request merged**